cypress/e2e/hub/collections-list.cy.ts

```diff
@@ -223,3 +218,9 @@ describe.skip('Collections List', () => {
     cy.deleteCollection(collectionName, namespace.name, repository);
   });
 });
+
+function actionClick(item: string, action: string) {
```

Comment:
Why not use `clickTableRowAction` ?

Reply:
It does not work here, not sure why, but we must now fully focus the project to make it finished.